### ISSUE TITLE
Not creating account?

### ISSUE THREAD

**Issue by CaliSDPerson** [tomlandia/fuckr#150]
When I click on "Sign Up", it just permanently gives the loading icon. This isn't a duplicate account from another device, is anybody else having this issue?

**Comment by RobbieTechie** [comment]
Did you get any error messages at all, or can you give more details, such as, is this an incident that can be duplicated?

**Comment by CaliSDPerson** [comment]
I just noticed the other thread, it's the same issue where it just hangs there. But none of the fixes in the other thread worked, I tried both 32 and 64.

**Comment by RobbieTechie** [comment]
When you create your account, is your password AT LEAST 6 characters? That is the bare minimum for a password on grindr.  If it is less, the app will sit there with the spinning circle.  It will return an error but the app doesn't show anything, similar to what you described.

Also after you create the account, uncheck the box at the top for photos only, that needs to be changed to not be checked by default.

**Comment by CaliSDPerson** [comment]
The password is 8 characters. There's no error, it just hangs.

**Comment by RobbieTechie** [comment]
Have you tried logging in with the actual app to test and see if the account was or was not successfully created?

**Comment by CaliSDPerson** [comment]
Even when trying to log in with an existing account it does the same thing. Is there perhaps a temp file that I can just clear as a fresh restart?

**Comment by RobbieTechie** [comment]
@CaliSDPerson Yes, you need to go to your AppData folder.  You can copy and paste this into your start menu search box:
`C:\Users\%username%\AppData\Local\Fuckr`
When that opens there is a folder called User Data, click on that, then delete it to clear everything from Fuckr.

Just a note, **_that will delete everything, including converversations_** then try to either create an account or log in again.

**Comment by Jd430** [comment]
I'm having the same problem.  I deleted that folder a while back and it was working fine.  Now it suddenly stopped.

**Comment by fm2000** [comment]
hi ... an account just created via the mobile client won't log in via the app — originally i wondered if it was because it had 6 character password (although my existing 6 character password accounts work fine on both) so changed it to 10 characters, but still won't work. retried and logs in fine via mobile, but i just get a grey "block/no smoking" type sign and it clears the email/password field to blank. all previous accounts continue working fine.

**Comment by RobbieTechie** [comment]
@fm2000 I have noticed, that the Fuckr app will sign in fine, then the next time you try to run it, it will keep you logged out, and you have to log in again.  Having 6 character passwords should not be an issue.  Have you tried closing the entire app down, and waiting a couple minutes without running the mobile version, and try logging back in.  If you are logged in on the mobile version at all, it will cause problems.  Maybe try shutting your phone or tablet off, then back on and don't use it before using Fuckr and see if that fixes anything.
